vscode-dotnet-runtime-library/src/Acquisition/VersionResolver.ts

```diff
@@ -38,15 +44,15 @@ export class VersionResolver implements IVersionResolver {
         }
     }
 
-    private resolveVersion(version: string, releases: ReleasesResult): string {
+    private resolveVersion(version: string, releases: ReleasesResult, runtimeVersion: boolean): string {
         this.validateVersionInput(version);
 
         const channel = releases.releasesIndex.filter((channelVal) => channelVal.channelVersion === version);
         if (isNullOrUndefined(channel) || channel.length !== 1) {
             throw new Error(`Unable to resolve version: ${version}`);
         }
-        const runtimeVersion = channel[0].latestRuntime;

```

Comment:
I forget, why again is it `channel[0]`? Aka what do the other indexes represent in the channels array?

Reply:
It's easiest to understand by looking at the structure of the file: https://github.com/dotnet/vscode-dotnet-runtime/blob/master/vscode-dotnet-runtime-library/src/test/mocks/mock-releases.json, each channel is associated with a major.minor .NET version. So here, since we filtered for the `channelVersion` we want (which would be `1.0`, `5.0`, `3.1`, etc) channel[0] is taking the first set of data associated with that major.minor version (there should only be one).

Reply:
Ahhh so we're depending on index 0 effectively being the "latest" Alright good to know